ed448/src/constants.rs

```diff
@@ -1,11 +1,15 @@
 use crate::*;
-use crate::{decaf::DecafPoint, Scalar};
+use crate::{
+    decaf::DecafPoint,
+    field::{ConstMontyType, FieldElement},
+    Scalar,
+};
 
 pub const DECAF_BASEPOINT: DecafPoint = DecafPoint(curve::twedwards::extended::ExtendedPoint {
-    X: TWISTED_EDWARDS_BASE_POINT.X,
-    Y: TWISTED_EDWARDS_BASE_POINT.Y,
-    Z: TWISTED_EDWARDS_BASE_POINT.Z,
-    T: TWISTED_EDWARDS_BASE_POINT.T,
+    X: FieldElement(ConstMontyType::new(&U448::from_be_hex("aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa955555555555555555555555555555555555555555555555555555555"))),
+    Y: FieldElement(ConstMontyType::new(&U448::from_be_hex("ae05e9634ad7048db359d6205086c2b0036ed7a035884dd7b7e36d728ad8c4b80d6565833a2a3098bbbcb2bed1cda06bdaeafbcdea9386ed"))),
+    Z: FieldElement(ConstMontyType::new(&U448::from_u64(1))),
+    T: FieldElement(ConstMontyType::new(&U448::from_be_hex("696d84643374bace9d70983a12aa9d461da74d2d5c35e8d97ba72c3aba4450a5d29274229bd22c1d5e3a6474ee4ffb0e7a9e200a28eee402"))),
 });
```

Comment:
I got this by taking the base point as [defined in the specification](https://www.rfc-editor.org/rfc/rfc9496.html#name-scalar-field:~:text=66666666%2066666666%2066666666%2066666666%2066666666%2066666666%2066666666%0A33333333%2033333333%2033333333%2033333333%2033333333%2033333333%2033333333) and doing:
```rust
println!("{:?}", CompressedDecaf(hex!("6666666666666666666666666666666666666666666666666666666633333333333333333333333333333333333333333333333333333333")).decompress());
```